extensions/libraries/redcore/translation/table.php

```diff
@@ -1206,4 +1206,25 @@ public static function setInstalledTranslationTables($option, $table, $contentEl
 			$translationTableModel->save($data);
 		}
 	}
+
+	/**
+	 * Method to purge contents of a translation table
+	 *
+	 * @param   string  $tableName  name of either the translations table or the table being translated I.E. #__content or #__content__rctranslations
+	 *
+	 * @return bool
+	 */
+	public static function purgeContentElement($tableName)
```

Comment:
you know that you have purge table by ID now? https://github.com/Mathewlenning/redCORE/blob/RSBTB-3392-Fatal-Error-When-Resetting-Translation-Tables/extensions/libraries/redcore/translation/table.php#L780